packages/compass-components/src/components/keyline-card.tsx

```diff
@@ -17,23 +20,74 @@ const keylineDarkThemeStyles = css({
   borderColor: palette.gray.dark2,
 });
 
+const lightHoverBoxShadow = `0 4px 20px -4px ${transparentize(
+  0.8,
+  palette.black
+)}`;
+
+const darkHoverBoxShadow = `0 4px 20px -4px ${transparentize(
+  0.3,
+  palette.black
+)}`;
+
+const clickableStyleLight = css`
+  cursor: pointer;
+
+  &:hover {
+    border: 1px solid ${palette.gray.light2};
+    box-shadow: ${lightHoverBoxShadow};
+  }
+
+  &:active {
+    box-shadow: none;
+  }
+`;
+
+const clickableStyleDark = css`
+  cursor: pointer;
+
+  &:hover {
+    box-shadow: ${darkHoverBoxShadow};
+  }
+`;
+
 const KeylineCard = forwardRef(function KeylineCard(
-  { className, ...props }: React.HTMLProps<HTMLDivElement>,
+  {
+    className,
+    contentStyle,
+    ...props
+  }: React.HTMLProps<HTMLDivElement> & {
+    /**
+     * Set the content style of the card (matches the interface of LeafyGreen cards).
+     *
+     * With `contentStyle="clickable"` the look and feel of the KeylineCard will resemble
+     * that of a button. Additionally, consistent focus ring handling and accessibility properties
+     * are added to the container.
+     */
+    contentStyle?: 'clickable' | 'none';
+  },
   ref: React.ForwardedRef<HTMLDivElement>
 ): React.ReactElement {
   const darkMode = useDarkMode();
+  const focusRingProps = useFocusRing();
 
-  return (
-    <div
-      ref={ref}
-      className={cx(
+  const allProps = mergeProps(
+    contentStyle === 'clickable'
+      ? { role: 'button', tabIndex: 0, ...focusRingProps }
+      : {},
+    {
+      className: cx(
```

Comment:
Related to the above question on box shadows, the clickable `Cards` from leafygreen have a box shadow that's always showing, and expanded when hovered. Would that make this more `clickable` looking? Fine as is too I reckon.

| not hovered | hovered |
| - | - |
| <img width="242" alt="Screenshot 2023-10-19 at 11 47 39 AM" src="https://github.com/mongodb-js/compass/assets/1791149/77222756-dbf4-4d13-b68c-0f2759a846e3"> | <img width="228" alt="Screenshot 2023-10-19 at 11 47 42 AM" src="https://github.com/mongodb-js/compass/assets/1791149/dce0c381-30b8-4a2f-a4e7-707afac9cb6a"> |

Reply:
mmm weird, I'm sure i copied the styles as they were in Card, I'll play around with that

Reply:
turns out i've ditched those styles by accident, added them back